Sources/Web3SwiftMpcProvider/EthTssAccountParams.swift

```diff
@@ -0,0 +1,27 @@
+public class EthTssAccountParams {
+    public let publicKey: String
+    public let factorKey: String
+    public let tssNonce: Int32
+    public let tssShare: String
+    public let tssIndex: String
+    public let selectedTag: String
+    public let verifier: String
+    public let verifierID: String
+    public let nodeIndexes: [Int]
+    public let tssEndpoints: [String]
+    public let authSigs: [String]
+    // Initializer
+    public init(publicKey: String, factorKey: String, tssNonce: Int32, tssShare: String, tssIndex: String, selectedTag: String, verifier: String, verifierID: String, nodeIndexes: [Int], tssEndpoints: [String], authSigs: [String]) {
```

Comment:
check with swiftlint
For now, to use swiftlint, -> uncomment the package and the plugin line code in package.swift 
then click build to run swift lint in the package.